src/lib/email/templates/VerificationEmail.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable unicorn/filename-case */
+import * as React from 'react'
+
+export interface VerificationEmailProps {
+  verificationUrl: string
+}
+
+export function VerificationEmail(props: VerificationEmailProps) {
+  const { verificationUrl } = props
+
+  return (
+    <div style={{ fontFamily: 'Arial, sans-serif', color: '#222' }}>
+      <h2>Verify your email address</h2>
+      <p>Please verify your email by clicking the link below:</p>
+      <p>
+        <a href={verificationUrl} style={{ color: '#2563eb' }}>
+          {verificationUrl}
+        </a>
+      </p>
+      <p>If you did not request this, you can ignore this email.</p>
+    </div>
```

Comment:
Add fallback for users who can’t click links

It’s common in email UX to include the raw URL as a fallback:

```tsx
<p>If the link above doesn't work, copy and paste this URL into your browser:</p>
<p style={{ wordBreak: 'break-all' }}>{verificationUrl}</p>
```

Some email clients may block links by default for security reasons